zhaquirks/ikea/tradfriplug.py

```diff
@@ -25,7 +25,7 @@
 from zhaquirks.ikea import IKEA, IKEA_CLUSTER_ID
 
 
-class TradfriPlug(CustomDevice):
+class TradfriPlug1(CustomDevice):
```

Comment:
Can being it braking some tests but its not running and i have not digging very deep in it.

Reply:
Looks like its only IKEA device that is having test is Starkvind so shall not braking it if i have understanding it OK.

Reply:
> Can being it braking some tests but its not running and i have not digging very deep in it.

Failing tests not seems to be related to PR.

Reply:
Thanks JC !!